.claude/skills/hns-lsel-curator/csa_refusal_test.sh

```
@@ -0,0 +1,144 @@
+    "decision": "approved",
+    "channel": "orchestrator AskUserQuestion"
+  }
+}
+JSON
+
+# Apply the refusal rule (a portable bash function mirroring the CSA doctrine in
+# the curator skill body). This is the rule M3's lsel-apply.sh will enforce
+# mechanically; here it is the test oracle.
+csa_refuse_check() {
+  local decision="$1"
+  local target blast sync
```

Shellcheck (0.11.0)

[warning] sync appears unused. Verify use (or export if used externally).

(SC2034)